org.gnome.Totem.json

```diff
@@ -26,6 +26,7 @@
         "--filesystem=xdg-download",
         /* Access DVDs */
         "--filesystem=/run/media",
+        "--filesystem=/media",
```

Comment:
Could you please add a reference for why discs are mounted in that location to the commit message?